src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/plugins/ConnectionPluginManager.java

```diff
@@ -58,6 +58,9 @@
     Runtime.getRuntime().addShutdownHook(new Thread(ConnectionPluginManager::releaseAllResources));
   }
 
+  /**
+   * Constructor.
```

Comment:
Include the parameters

Reply:
Excluding parameters for constructors that only require loggers. I am tempted to remove these single line javadoc too as it does not provide value.